src/fair/stats/__init__.py

```diff
@@ -566,3 +567,70 @@ def sample(self, n: int = 1) -> np.ndarray:
             samples.append(mBeta.sample())
 
         return np.vstack(samples)
+
+
+class GOF(mBeta):
+    """Goodness of fit test for mBeta distribution as described in section 2
+    https://arxiv.org/pdf/2003.06684
+    """
+
+    def __init__(self, null: mBeta, alt: mBeta) -> None:
+        """
+        Args:
+            null (mBeta): null mBeta distribution
+            alt (mBeta): alternative mBeta distribution
+        """
+        self.null = null
+        self.alt = alt
+
+    def _null_ecdf(self, n_samples: int = 100, t_samples: int = 100) -> ECDFResult:
```

Comment:
I found this confusing, at first I thought you meant null as in empty, maybe writing null hypothesis instead of just null makes it a bit more clear?

Reply:
Ok gotcha. It's not the null hypothesis, but rather the distribution under that hypothesis, so I changed it to `null_dist` and `alt_dist`.